src/coreclr/src/inc/readytoruninstructionset.h

```diff
@@ -0,0 +1,35 @@
+
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// DO NOT EDIT THIS FILE! IT IS AUTOGENERATED
+#ifndef READYTORUNINSTRUCTIONSET_H
+#define READYTORUNINSTRUCTIONSET_H
+enum ReadyToRunInstructionSet
+{
+    READYTORUN_INSTRUCTION_Sse=1,
```

Comment:
It seems strange that we are mixing x86 and ARM instruction sets in here where we separate them in other parts of the runtime.

Reply:
The ReadyToRun constants are generally defined in an architecture independent way, and this just follows that convention.

Reply:
How is this going to expand beyond 32-bits?

Reply:
These constants are simply increasing numbers. The planned encoding for this is actually as an array of raw numbers so there is no limit to 32 different instruction sets at all, the limit is more like 2^30 different instruction sets supported, which is definitely enough.